.github/workflows/arm64.yml

```diff
@@ -72,6 +72,7 @@ jobs:
         uses: Alfresco/alfresco-build-tools/.github/actions/docker-dump-containers-logs@v1.41.0
         if: failure() && steps.setup-env.outcome == 'failure'
       - name: "Run tests"
+        id: run-tests
```

Comment:
Let's remove this unused id, @mikolajbrzezinski will take care of improving the Docker logs in his PR.